tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/genitals.tsx

```diff
@@ -79,6 +83,7 @@ export const testicles_color: Feature<string[]> = {
 
 export const testicles_emissive: Feature<boolean[]> = {
   name: 'Testicles Emissives',
+  description: 'Emissive parts/layers glow in the dark.',
```

Comment:
```suggestion
  description: 'Emissive parts glow in the dark.',
```
Could be shortened even more. We don't need to be educating players about emissive layers in these tooltips